app/src/main/java/org/dhis2/usescases/teiDashboard/dashboardfragments/indicators/AnalyticsAdapter.kt

```diff
@@ -54,43 +56,9 @@ class AnalyticsAdapter(val context: Context) : ListAdapter<AnalyticsModel, Recyc
 
     override fun onBindViewHolder(holder: RecyclerView.ViewHolder, position: Int) {
         when (holder) {
-            is IndicatorViewHolder -> holder.bind(items[position] as AnalyticsModel.IndicatorModel)
-            is ChartViewHolder -> holder.bind(items[position] as AnalyticsModel.ChartModel)
-            is SectionTitleViewHolder -> holder.bind(items[position] as AnalyticsModel.SectionTitle)
+            is IndicatorViewHolder -> holder.bind(getItem(position) as IndicatorModel)
+            is ChartViewHolder -> holder.bind(getItem(position) as ChartModel)
+            is SectionTitleViewHolder -> holder.bind(getItem(position) as SectionTitle)
         }
     }
-
-    override fun getItemCount() = items.size
-
-    fun setIndicators(indicators: List<AnalyticsModel.IndicatorModel>) {
-        val title = context.getString(R.string.dashboard_indicators)
-
-        items.removeAll {
-            it is AnalyticsModel.IndicatorModel ||
-                it is AnalyticsModel.SectionTitle && it.title == title
-        }
-        if (indicators.isNotEmpty()) {
-            items.addAll(0, listOf(AnalyticsModel.SectionTitle(title)) + indicators)
-        }
-
```

Comment:
This was used for add section tittles, if we want to keep them we need to add SectionTitles to adapter anywhere, otherwise SectionTitle class is useless

Reply:
The current design does not have sections.
We decided to add the required methods for the future once the final design is ready. This is why i've removed it for now.